x-pack/legacy/plugins/siem/public/pages/detection_engine/detection_engine.tsx

```diff
@@ -65,8 +66,7 @@ export const DetectionEngineComponent = React.memo(() => {
                 </EuiPanel>
 
                 <EuiSpacer />
-
-                <SignalsTable />
+                <GlobalTime>{({ to, from }) => <SignalsTable from={from} to={to} />}</GlobalTime>
```

Comment:
👍

Reply:
Yeah I ended up yanking it out as the histogram needs it as well. Once we update our redux/hooks libs, I wonder if it'll be better/cleaner to just wire up the components individually? Will think about that one.

Reply:
No please do not do it individually, or it will break everything